src/compression.jl

```diff
@@ -285,8 +293,8 @@ function randcompress(A::AbstractMatOrLinOp{T}, rcl::ClusterTree, ccl::ClusterTr
 
   # compute initial sampling
   k = kest; r = opts.noversampling;
-  Ωcol = randn(n, k+r)
-  Ωrow = randn(m, k+r)
+  Ωcol = randn(T,n, k+r)
```

Comment:
is the inclusion of T here correct? I am not sure how the sampling matrix in randomized SVD needs to be for complex numbers?

Reply:
Good point, I will check that.